tests/e2e/Exclude_Mutations_By_Regex/infection.json

```diff
@@ -0,0 +1,31 @@
+{
+    "timeout": 25,
+    "source": {
+        "directories": [
+            "src"
+        ]
+    },
+    "logs": {
+        "summary": "infection.log"
+    },
+    "tmpDir": ".",
+    "mutators": {
+        "global-ignoreSourceCodeByRegex": [
+            ".*hello.*"
+        ],
+        "PublicVisibility": {
+            "ignore": [
+                "App\\*"
+            ],
+            "ignoreSourceCodeByRegex": [
+                ".*getString.*"
+            ]
+        },
+        "MethodCallRemoval": {
+            "ignoreSourceCodeByRegex": [
+                "Assert::.*",
```

Comment:
Shouldn't we dog-food this feature already?

Reply:
As a non-native English speaker, may I ask you what does this mean?

Reply:
Sure. I'm slightly surprised to find that there's even [a Wikipedia article for the term](https://github.com/symfony/symfony/pull/38191).

> Eating your own dog food or dogfooding is the practice of an organization using its own product.

Reply:
> Shouldn't we dog-food this feature already?

if we need to - we certainly can. I didn't do any analysis for this though

Reply:
It can certainly wait.